src/main/java/frc/robot/RobotContainer.java

```diff
@@ -47,9 +48,9 @@ private void configureButtonBindings() {
     // Define commands here
     DriveWithJoystick defaultDriveCommand = new DriveWithJoystick(
         drive,
-        controls::getLeftDriveY,
-        controls::getRightDriveX);
-
+        () -> controls.getLeftDriveY(),
+        () -> controls.getRightDriveX());
+    
```

Comment:
Why this change? Using the lambda is slightly less efficient as it just wraps the function instead of calling it directly

Reply:
Didn't realize the efficiency aspect of it, I thought that doing this would be clearer to others what this part is doing, but we can change it back